Shifty.App/Components/ProductManager.razor

```diff
@@ -5,17 +5,22 @@
 @using Shifty.Api.Generated.AnalogCoreV2
 @using Shared
 @using LanguageExt.UnsafeValueAccess
+@using Components
 @inject ISnackbar Snackbar
 @inject IProductService ProductService
 
-<MudPaper Elevation="15" Style="margin: 40px auto; border-radius: 5px; max-width: 90%;">
+<style>
+    .mud-table-cell {
+        padding: 0.5em 0.75em !important;
+    }
+</style>
+<MudPaper Elevation="15" Style="margin: 2em; border-radius: 5px;">
     @if (_loading)
     {
         <MudContainer Style="width: 100%; display: flex;">
             <LoadingIndicator Height="400px" />
         </MudContainer>
     }
-    <MudSwitch @bind-Checked="_showNonvisible" Label="Show non-visible products" Color="Color.Primary" Class="mt-5 ml-2" />
```

Comment:
I thought you wanted this feature? 😅

I don't really mind the feature or not - the filters can probably do it for us anyway.

However: Removing this line removes the UI element but not the logic. If we remove this feature, we should also remove the field on line 129 and the QuickerFilter on line 34 (in the MudDataGrid object).